source/_integrations/fritz.markdown

```diff
@@ -56,7 +56,7 @@ Reboot the router.
 
 | Service data attribute | Optional | Description                                                                                                    |
 | ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------- |
-| `entity_id`            | no       | Only act on a specific  router                                                                                 |
+| `device_id`            | no       | Only act on a specific  router                                                                                 |
```

Comment:
How is this related?

Reply:
Current services are using `device_id` . This is wrong in the docs
![image](https://user-images.githubusercontent.com/18382402/147513250-d960d137-0a51-46a9-a90b-e31dffb671e4.png)

Reply:
Might, be, but how is that related to the parent PR of this documentation PR?

Reply:
I thought I fix this issue as I saw it...

Reply:
Cool, but isn't related to this PR?

Reply:
Actually not to https://github.com/home-assistant/core/pull/62892 if you mean...

Reply:
I will split this into another one tomorrow